python/llm/src/bigdl/llm/optimize.py

```diff
@@ -14,11 +14,42 @@
 # limitations under the License.
 #
 
+import torch
+import os
+import json
 from .transformers import ggml_convert_low_bit
 from bigdl.llm.ggml.quantize import ggml_tensor_qtype
 from bigdl.llm.utils.common import invalidInputError
 
 
+# Simulate the Hugging Face format
+PYTORCH_MODEL_NAME = "pytorch_model.bin"
+CONFIG_NAME = "bigdl_config.json"
+
+
+def _save_low_bit(self, save_dir, *args, **kwargs):
+    os.makedirs(save_dir, exist_ok=True)
+    model_path = os.path.join(save_dir, PYTORCH_MODEL_NAME)
+    torch.save(self.state_dict(), model_path, *args, **kwargs)
+    with open(os.path.join(save_dir, CONFIG_NAME), "w") as json_file:
+        json.dump(self._bigdl_config, json_file)
+
+
+def load_low_bit(model, model_path):
```

Comment:
Need to properly handle error conditions:
- Not a directory
- No config file
- Not low bit model
- No model file
- ...

Reply:
sure we do